de-identification-app/src/main/resources/application.properties

```diff
@@ -1,11 +1,11 @@
 ###############################################################################
-# (C) Copyright IBM Corp. 2016,2020
+# (C) Copyright IBM Corp. 2016,2021
 #
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################################
  
-
 logging.level.root=INFO
```

Comment:
Should INFO be the default ? You'd think that this is going to log PHI at one place or another

Reply:
INFO is what we've been testing primarily.  It seems reasonable.  The problems we've seen have been at DEBUG level except for the new third-party code.   We can change it to WARN if you are more comfortable with that?

Reply:
DEBUG would be really bad as default, yes. If you look at the logs at INFO after a few unit tests ran for example and it seems reasonable, we can simply document that for production we recommend Warn or Error. If we don't trust INFO anymore as default then we could go to WARN in that file

Reply:
For now, I would say we don't have a reason not to trust INFO.